sdk/maintenance/Azure.ResourceManager.Maintenance/tests/Scenario/MaintenanceConfigurationTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Maintenance.Models;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Maintenance.Tests
+{
+    public sealed class MaintenanceConfigurationTests : MaintenanceManagementTestBase
+    {
+        private SubscriptionResource _subscription;
+        private ResourceGroupResource _resourceGroup;
+        private MaintenanceConfigurationCollection _configCollection;
+
+        public MaintenanceConfigurationTests(bool isAsync) : base(isAsync) //, RecordedTestMode.Record)
+        { }
+
+        [SetUp]
+        public async Task Setup()
+        {
+            _subscription = await Client.GetDefaultSubscriptionAsync();
+
+            _resourceGroup = await CreateResourceGroup(
+               _subscription,
+               "Maintenance-RG-",
+               Location);
+
+            _configCollection = _resourceGroup.GetMaintenanceConfigurations();
+        }
+
+        [TearDown]
+        public async Task TearDown()
+        {
+            await _resourceGroup.DeleteAsync(WaitUntil.Completed);
+        }
+
+        [RecordedTest]
+        public async Task DeleteTest()
```

Comment:
We do not need to add the word `Test` on the method name
```suggestion
        public async Task Delete()
```
If you change this method name, you do not need to record the test cases again, just rename the recording file correspondingly.